Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -109,7 +107,6 @@ namespace Multiplayer
         }
         else if (AZ::BehaviorContext* behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
         {
-            behaviorContext->Class<HostId>();
```

Comment:
Tagging @AMZN-Gene to make sure there aren't any scripting concerns with this. Seems fine to me.

Reply:
Yeah, that used to be a totally abstract hostId assigned at boot time. It doesn't make sense to keep that abstraction anymore, using a fully addressable identifier like IP address works out much better, and in fact this almost immediately cleaned up a bunch of code.